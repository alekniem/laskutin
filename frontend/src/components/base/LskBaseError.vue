<template>
    <div class="red--text font-weight-bold">
        <slot />
    </div>
</template>

<script>
export default {
    name: "LskBaseError",
};
</script>
