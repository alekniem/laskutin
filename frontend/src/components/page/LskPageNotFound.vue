<template>
    <LskBaseError>
        {{ "Page not found" | trans }}
    </LskBaseError>
</template>

<script>
export default {
    name: "LskPageNotFound",
};
</script>
