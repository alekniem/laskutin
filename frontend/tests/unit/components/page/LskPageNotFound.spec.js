import flushPromises from "flush-promises";
import LskBaseError from "@/components/base/LskBaseError";
import LskPageNotFound from "@/components/page/LskPageNotFound";
import { createWrapper } from "UT/lsk_test_helper";

const stubs = {
    LskBaseError,
};

describe("LskPageNotFound", () => {
    it("should match snapshot", async () => {
        const wrapper = createWrapper(LskPageNotFound, {
            stubs,
        });
        await flushPromises();
        expect(wrapper.html()).toMatchSnapshot();
    });
});
