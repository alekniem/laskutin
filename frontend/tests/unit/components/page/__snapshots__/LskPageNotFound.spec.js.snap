// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LskPageNotFound should match snapshot 1`] = `
<div class="red--text font-weight-bold">
  Page not found
</div>
`;
